recipe_scrapers/emmikochteinfach.py

```diff
@@ -8,7 +8,7 @@ def host(cls):
         return "emmikochteinfach.de"
 
     def author(self):
-        return self.schema.author()
+        return "Christiane Emma"
```

Comment:
I think we should restore schema-based retrieval here, unless there's a reason not to; generally if the page does contain authorship information in standard formats (schema.org / meta author) then that seems like an indication of the preferred way they'd like to be credited - and those page elements should be things that they can adjust if they decide on a change for that.

```suggestion
        return self.schema.author()
```